test/support/helloworld.pb.ex

```diff
@@ -1,31 +1,56 @@
 defmodule Helloworld.HelloRequest do
-  use Protobuf
+  @moduledoc false
```

Comment:
Regenerated. I used the general command for recreation, but if I need to try again with other args, I can do so.